### ISSUE TITLE
Bug in Amp Lightbox Gallery where it skips an image

### ISSUE THREAD

**Issue by maxpelic** [ampproject/amphtml#21559]
## What's the issue?

On my Google Pixel 3, when scrolling through the AMP lightbox gallery, it skips an image... I swipe over to the next image, and while swiping everything seems fine - I can see the current image moving over and the next image coming in. But when I'm done swiping, it just skips to the image after the one I was swiping to

## How do we reproduce the issue?

Here's an example of the issue: https://codepen.io/maxpelic/pen/KEEQMz?editors=1000
It only works if you swipe the image most of the way over before releasing 

1. Open the link (I've only tested it on my Google Pixel 3)
2. click on an image to open the lightbox gallery
3. swipe over to the next picture

## What browsers are affected?

Google Pixel 3 - Google Chrome (stable release)

## Which AMP version is affected?

Here's the version I'm using: 1903141753530

**Comment by aghassemi** [comment]
Thanks for the great repro sample on codepen @maxpelic .

To @sparhami to triage.

**Comment by sparhami** [comment]
Since it can be a bit difficult to fix bugs with the current carousel, we should take a look at addressing this by making the lightbox use `<amp-base-carousel>` (#21568). There are a few more things we need before we can move forward with the change however.

**Comment by archon810** [comment]
Any updates here? Pretty sure we're affected by this bug https://github.com/android-police/androidpolice-public/issues/100.

**Comment by sparhami** [comment]
We have a new version of amp-carousel which will fix this, but haven't changed the lightbox gallery over to use it yet. We plan on changing over in the next few weeks.

**Comment by archon810** [comment]
Any updates here?

**Comment by archon810** [comment]
Hi, asking again about the fix for this issue.

**Comment by archon810** [comment]
Is this fixed in v0.2 as well, like https://github.com/ampproject/amphtml/issues/22608?

**Comment by nainar** [comment]
Yes this should be fixed if using carousel v0.2, please let us know if you have any issue by filing a new issue
